### DISCUSSION TITLE
Black Rib Armor Trim

### DISCUSSION THREAD

**Discussion by ParkR15MineR** [BedrockTweaks/Files#271] [Ideas]
New idea! When you select the black Netherite bricks, the Rib Armor Trim should also turn black, since you need Netherite bricks to duplicate it

![IMG_0242](https://github.com/user-attachments/assets/9bf0da32-4aeb-47cf-819d-2f7d953e3bed)

**Comment by SquatchHunter** [comment]
I feel like the armour trim items could have a nice visual upgrade done to them, but for now this is a "Community Pack" and probably wont be looked into till after Addons release
